spec/datadog/tracing/contrib/active_record/instantiation_spec.rb

```diff
@@ -12,9 +12,6 @@
   let(:article) { Article.first }
 
   before do
-    if Gem::Version.new(ActiveRecord::VERSION::STRING) < Gem::Version.new('4.2')
-      skip 'ActiveRecord instantiation events were added in Rails 4.2'
-    end
```

Comment:
I did spot that the public documentation lists Rails 4.0, but we're only testing with 4.2 and so indeed this is not needed.

@marcotc I think we should update the public docs to list 4.2 instead of 4.0?